packaging/server_config.json.template

```diff
@@ -10,13 +11,18 @@
             "shared_memory_size_in_bytes": 5000000,
             "eviction_age_in_seconds": 3600
         },
+        "dns_cache_clearer_sleep_time_in_seconds": 600,
         "hostname_cache": {
             "shared_memory_size_in_bytes": 2500000,
             "eviction_age_in_seconds": 3600
         },
+        "hostname_cache_clearer_sleep_time_in_seconds": 600,
         "maximum_size_for_single_buffer_in_megabytes": 32,
         "maximum_size_of_delay_queue_in_bytes": 0,
         "maximum_temporary_password_lifetime_in_seconds": 1000,
+        "migrate_delay_server_sleep_time_in_seconds": 5,
+        "migrate_delay_server_socket_receive_timeout_in_milliseconds": 5000,
+        "migrate_delay_server_socket_send_timeout_in_milliseconds": 500,
```

Comment:
This made me wonder if we should include a "delay_server" section in the configuration since we now have the following configurations:
 - delay_rule_executors
 - delay_server_sleep_time_in_seconds
 - maximum_size_of_delay_queue_in_bytes
 - migrate_delay_server_sleep_time_in_seconds
 - migrate_delay_server_socket_receive_timeout_in_milliseconds
 - migrate_delay_server_socket_send_timeout_in_milliseconds
 - number_of_concurrent_delay_rule_executors

Reply:
I suppose some of these are configurations for the delay server itself and some are configurations for machinery around the delay server, i.e. delay server migration... Anyways, something to consider.

Reply:
That's a possibility. The main challenge is upgrades and expectations of administrators.

There's also the discussion of a flat configuration vs a configuration with nesting.

Reply:
Very true. Keeping the flat structure for these options in particular seems fine/preferred, so I'll leave it how it is.